src/block/server/genode/block_dispatcher.h

```diff
@@ -22,12 +22,12 @@ namespace Block
                     void *callback = nullptr);
             void finalize();
             void announce();
-            __attribute__((annotate("ada"))) void dispatch();
-            char *label_content();
-            Genode::uint64_t label_length();
-            Genode::uint64_t session_size();
-            void session_accept(void *session);
-            bool session_cleanup(void *session);
+            __attribute__((annotate("ada"))) void dispatch(void *dcap);
```

Comment:
Yes it is, we don't necessarily need it anymore. I could remove it if you want to although I have to say that the majority of this code (and its structure) is basically a leftover from Cappulada and it would require some more effort to clean it from that.

Reply:
True. I guess all the void pointers are one such thing? Please remove this annotation and create a clean-up ticket in 1.99 to address those structural issues (with a task list what we want to change).

Reply:
I opened https://github.com/Componolit/ada-interfaces/issues/76. The list might not be complete yet.